### DISCUSSION TITLE
How can I make a selective test run in AllureTestOps with framework playwright+javascript?

### DISCUSSION THREAD

**Discussion by blamaniq** [allure-framework/allure2#1825] [Questions & Support] [answered]
How can I make a selective test run in AllureTestOps with framework playwright+javascript? Do you have a template of selective run on playwright?

**Comment by cheshi-mantu** [comment, marked as answer]
@blamaniq 
Hello there
Unfortunately, this does not exist for PlayWright at the moment due to the framework limitations.
See here: https://github.com/microsoft/playwright/issues/17037
